spec/integration/client_construction_spec.rb

```diff
@@ -100,6 +100,27 @@
         expect(server.description).to be_primary
       end
     end
+
+    # This test requires a PSA deployment. The port number is fixed for our
+    # Evergreen/Docker setups.
+    context 'when directly connecting to arbiters' do
+      let(:options) do
+        SpecConfig.instance.test_options.tap do |opt|
+          opt.delete(:connect)
+          opt.update(direct_connection: true)
+        end
+      end
+
+      let(:client) do
+        new_local_client(['localhost:27019'], options)
+      end
+
+      let(:response) { client.command(ismaster: 1).documents.first }
```

Comment:
```suggestion
      let(:response) { client.command(hello: 1).documents.first }
```

Reply:
This is not going to work on 3.6 is it?

Reply:
Oh, my bad! Just ignore the suggestion, sorry.